app/i18n/it/sub.php

```diff
@@ -178,32 +178,32 @@
 		'title' => 'Importa / esporta',
 	),
 	'menu' => array(
-		'add' => 'Add a feed or category',	// TODO
+		'add' => 'Aggiungi un feed o una categoria',
 		'import_export' => 'Importa / esporta',
-		'label_management' => 'Label management',	// TODO
+		'label_management' => 'Gestione etichette',
 		'stats' => array(
 			'idle' => 'Feeds non aggiornati',
 			'main' => 'Statistiche principali',
 			'repartition' => 'Ripartizione articoli',
 		),
 		'subscription_management' => 'Gestione sottoscrizioni',
-		'subscription_tools' => 'Subscription tools',	// TODO
+		'subscription_tools' => 'Strumenti di sottoscrizione',
 	),
 	'tag' => array(
-		'name' => 'Name',	// TODO
-		'new_name' => 'New name',	// TODO
-		'old_name' => 'Old name',	// TODO
+		'name' => 'Nome',
+		'new_name' => 'Nuovo nome',
+		'old_name' => 'Vecchio nome',
 	),
 	'title' => array(
 		'_' => 'Gestione sottoscrizioni',
-		'add' => 'Add a feed or category',	// TODO
-		'add_category' => 'Add a category',	// TODO
-		'add_dynamic_opml' => 'Add dynamic OPML',	// TODO
-		'add_feed' => 'Add a feed',	// TODO
-		'add_label' => 'Add a label',	// TODO
-		'delete_label' => 'Delete a label',	// TODO
+		'add' => 'Aggiungi un feed o una categoria',
+		'add_category' => 'Aggiungi una categoria',
+		'add_dynamic_opml' => 'Aggiungi OPML dinamico',
+		'add_feed' => 'Aggiungi un feed',
+		'add_label' => 'Aggiungi un\'etichetta',
```

Comment:
```suggestion
		'add_label' => 'Aggiungi un’etichetta',
```

Favour the typographic apostrophe `’` instead, which also has the advantage of not having to be prefixed by a back-slash, in addition of being nicer